osu.Game/Updater/UpdateManager.cs

```diff
@@ -44,6 +45,11 @@ protected override void LoadComplete()
             config.Set(OsuSetting.Version, version);
         }
 
+        public virtual void CheckForUpdate()
+        {
+            Logger.Log("CheckForUpdate was called on the base class (UpdateManager)", LoggingTarget.Information);
```

Comment:
Why is this being logged at all?

Reply:
I felt this is the right thing to do. You don't want to silently do nothing, because you missed something or expected something else.

Reply:
Doesn't seem very useful. There is no practical difference to the user between doing nothing and logging this (default log level is verbose, so it won't show up anywhere unless you manually check the log). I'd argue that in the cases that would log this, the button should be either disabled or not even visible to users.

I'm pretty sure the only platform that uses the base `UpdateManager` directly is iOS, on which the update button is probably pretty useless anyway in the long run due to the app store existing (I presume testflight also handles app updates, but I wouldn't know).

Reply:
I'm going to keep the log message for purely debugging purposes. But I'm going to hide the button if we're having the base class.